### ISSUE TITLE
Image characterization not correct for `:inline` queue

### ISSUE THREAD

**Issue by qjhart** [UCDavisLibrary/ucd-hyku#4]
### Descriptive summary

If you set the queuing behavior to `:inline`, then while importing new items to the system, the images do not show up as having been characterized.  Even though they have been.

### Expected behavior

You would expect the image data to become immediately become available.  

### Actual behavior

You would expect the image data to become immediately become available.  Instead the image is identified as not characterized.  Any further edits, (fro example , just navigating to the `Edit->permissions`  tab and saving the current permissions, will fix this issue.

### Steps to reproduce the behavior

1. Start fresh,   
``` bash
alias dc='docker-compose'
git diff

diff --git a/config/environments/production.rb b/config/environments/production.rb
index 28f9b0a..747274a 100644
--- a/config/environments/production.rb
+++ b/config/environments/production.rb
@@ -56,8 +56,9 @@ Rails.application.configure do
   # config.cache_store = :mem_cache_store

   # Use a real queuing backend for Active Job (and separate queues per environment)
-  require 'active_job/queue_adapters/better_active_elastic_job_adapter'
-  config.active_job.queue_adapter     = Settings.active_job.queue_adapter
+  #require 'active_job/queue_adapters/better_active_elastic_job_adapter'
+  #config.active_job.queue_adapter     = Settings.active_job.queue_adapter
+  config.active_job.queue_adapter = :inline
   # config.active_job.queue_name_prefix = "hyku_#{Rails.env}"

   config.action_mailer.perform_caching = false
```
1.` dc build`
1. `dc down -v; dc up -d`.
1.  Log in create a tenant, create a user, add a work. 
1.  Work is mis-represented as `Not characterized`
1.  Use 'Edit->permissions' to touch file, and things look okay.

This image shows the results after upload
![image](https://user-images.githubusercontent.com/344953/27205630-cdc80a20-51e6-11e7-9441-fc5321fb8eb5.png)

This image shows the file status.

![image](https://user-images.githubusercontent.com/344953/27205668-0796dba0-51e7-11e7-9f13-edaea4ed500d.png)

Even though, and investigation of the fedora backend seems to indicate that they have been properly indexed.

![image](https://user-images.githubusercontent.com/344953/27205766-a00710b2-51e7-11e7-8d78-f9dd07f75249.png)

Any sort of update to the file, will bring in the characterization.

![image](https://user-images.githubusercontent.com/344953/27297515-f81e4802-54d9-11e7-8af6-95be93eb4748.png)

**Comment by mjgiarlo** [comment]
@qjhart Good find! Might you be inclined to troubleshoot why this method would be returning `false` if the characterization metadata is present in Fedora? https://github.com/samvera/hyrax/blob/master/app/presenters/hyrax/characterization_behavior.rb#L22-L24 When this method returns `false`, the `not yet characterized` message is printed out.

This could be a bug we ought to fix in Hyrax.

**Comment by qjhart** [comment]
@mjgiarlo, yeah, we will continue to pursue.  Not the best rails programmers here :)
